pass-nihms-loader/nihms-pass-client/src/main/java/org/eclipse/pass/client/nihms/cache/GrantIdCache.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2023 Johns Hopkins University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.pass.client.nihms.cache;
+
+import java.util.HashMap;
+
+/**
+ * Caches awardNumber and grantId combination for easy lookup
+ *
+ * @author Karen Hanson
+ */
+public class GrantIdCache {
```

Comment:
I don't see the purpose of these xxxCache classes.  They are thin wrappers around a Map for caching purposes.  I would ask why not just use a Map in `NihmsPassClientService`, etc.  If the Map really needs to be synchronized, `Collections.synchronizedMap()` or `ConcurrentHashMap` could be used.

Reply:
I believe we created a ticket to address this a while back: https://github.com/eclipse-pass/main/issues/566

Do you want me to address it in that ticket or add it to the pass-support nihms cleanup ticket? 

https://github.com/eclipse-pass/main/issues/602

Reply:
I'd say to probably combine 566 into 602, just so all items are in one ticket.